core/src/components/button/button.ionic.scss

```diff
@@ -0,0 +1,239 @@
+@import "./button";
+@import "./button.ionic.vars";
+
+// Ionic Button
+// -------------------------------------------------------------------------------
+
+:host {
+  --border-radius: #{$button-ionic-default-border-radius};
+  --padding-top: #{$button-ionic-default-padding-top};
+  --padding-end: #{$button-ionic-default-padding-end};
+  --padding-bottom: var(--padding-top);
+  --padding-start: var(--padding-end);
+
+  position: relative;
+
+  min-height: #{$button-ionic-default-min-height};
+
+  font-size: #{$button-ionic-default-font-size};
+
+  // Target area
+  &::after {
+    position: absolute;
+
+    top: 50%;
+    /* stylelint-disable-next-line property-disallowed-list */
+    right: 0;
+    bottom: 0;
+    /* stylelint-disable-next-line property-disallowed-list */
+    left: 0;
+
+    height: 100%;
+    min-height: pxToRem(48);
+    
+    transform: translateY(-50%);
+  
+    content: "";
+  
+    cursor: pointer;
+
+    z-index: 1;
+  }
+
+  ::slotted(ion-icon[slot=start]) {
+    margin-inline-end: pxToRem(8);
```

Comment:
This feedback applies to all usages of padding/margin used in the implementation. 

Let's use the `@include margin` and `@include padding` mixins.